LoopUI/Views/DeviceStatusHUDView.swift

```diff
@@ -22,7 +22,7 @@ import LoopKitUI
     @IBOutlet public weak var progressView: UIProgressView! {
         didSet {
             progressView.isHidden = true
-            progressView.tintColor = .systemPurple
+            progressView.tintColor = .systemGray
```

Comment:
Just double-checking that the change from purple to gray here was intentional

Reply:
yes. I originally thought the purple colour was a Tidepool accent colour, but instead it is a specific glucose colour. This progress bar is the life bar of either the pump or CGM and the colour will be related to the type of device.